lib/pf/provisioner/jamf.pm

```diff
@@ -250,12 +265,12 @@ sub parse_device_information {
     my $logger = get_logger;
 
     my $json = decode_json($device_information);
-
+    
```

Comment:
remove space

Reply:
I was trying to make some space for @lzammit returning back to work next week. Probably gained a few pounds in the last 5 weeks